roles/mirror_from_directory/README.md

```diff
@@ -0,0 +1,34 @@
+# Mirror from file role
+
+A role that mirror operators from a local directory into a container images registry using the `oc-mirror` plugin.
+
+## Parameters
+
+Name                   | Required | Default                                       | Description
+---------------------- | -------- | --------------------------------------------- | ------------
+mfd_local_registry     | Yes      |                                               | Target registry
+mfd_operators_dir      | Yes      |                                               | Directory containing the tarball files generated by oc-mirror
+mfd_delete_workspace   | No       | True                                          | Deletes the `oc-mirror` workspace generated during the mirroring
+
+## Outputs
+
+The role copies a generated Image Sources manifest and CatalogSources files into a temporary directory. The files can bee access those files via following variables:
```

Comment:
The files can bee access those files via following variables -> The files can access to those files via the following variables

Reply:
Thanks, I reworded the paragraph.